perun-base/src/test/resources/test-schema.sql

```diff
@@ -1638,6 +1638,16 @@ create table consent_attr_defs (
 	constraint consentatt_consent_fk foreign key (consent_id) references consents(id)
 );
 
+-- APP_NOTIFICATIONS_SENT - sent applications notifications, used only for APP_CREATED_VO_ADMIN type for now
+create table app_notifications_sent (
+	app_id int not null,
+	notification_type varchar not null,
```

Comment:
Wouldn't it be better to use an enum to represent the type?

Reply:
Yes, thanks for the suggestion.